isis/tests/FunctionalTestsCsminit.cpp

```diff
@@ -266,3 +277,64 @@ TEST_F(CSMPluginFixture, CSMinitFails) {
   // Expect a failure due to being unable to construct any model from the isd
   EXPECT_ANY_THROW(csminit(options));
 }
+
+
+TEST_F(DefaultCube, CSMinitSpiceCleanup) {
```

Comment:
This is possibly obvious, but could you put in a comment that this test (and the next) use the DefaultCube fixture because it is a spiceinited cube?